### ISSUE TITLE
Error when enabling scalafix in multi project build on versions > 0.3.1

### ISSUE THREAD

**Issue by artogai** [typelevel/sbt-tpolecat#118]
Hello. I have encountered compile error when enabling scalafix on multi project build on versions > 0.3.1. 

Error:
```
[error] bad option: -P:semanticdb:targetroot:${project_dir}/module1/target/scala-2.13/meta
```

build.sbt
```
import _root_.io.github.davidgregory084.ScalacOptions

ThisBuild / scalaVersion := "2.13.8"
ThisBuild / semanticdbEnabled := true
ThisBuild / semanticdbVersion := scalafixSemanticdb.revision

lazy val commonSettings = 
  Seq(
    Compile / tpolecatScalacOptions := Set()
  )

lazy val module1 = (project in file("module1"))
  .settings(name := "module1")
  .settings(commonSettings)

lazy val module2 = (project in file("module2"))
  .settings(name := "module2")
  .settings(commonSettings)

lazy val root = (project in file("."))
  .settings(name := "root")
  .aggregate(module1, module2)

Global / onChangedBuildSource := ReloadOnSourceChanges
```

Minimal example: 
https://github.com/artogai/tpolecat-scalameta

**Comment by artogai** [comment]
I'm wondering if this is related to re-assigning `scalacOptions` with `:=`

**Comment by DavidGregory084** [comment]
Yes I think you are right, I think this might be another manifestation of #102 😢 I'm pretty baffled as to why that results in passing an invalid compiler option with an unresolved substitution in it though!

**Comment by DavidGregory084** [comment]
Hi @artogai I think this should be fixed by the changes in 0.4.2 - can you possibly try it out and report back?

**Comment by artogai** [comment]
Hello @DavidGregory084 . Yes, it works. Thank you!
